src/estimagic/inference/bootstrap.py

```diff
@@ -60,46 +67,178 @@ def bootstrap(
         batch_evaluator=batch_evaluator,
     )
 
-    out = bootstrap_from_outcomes(
-        data, outcome, estimates, ci_method=ci_method, alpha=alpha, n_cores=n_cores
-    )
+    base_outcome = outcome(data)
+    out = bootstrap_from_outcomes(base_outcome, bootstrap_outcomes)
 
     return out
 
 
-def bootstrap_from_outcomes(
-    data, outcome, bootstrap_outcomes, *, ci_method="percentile", alpha=0.05, n_cores=1
-):
-    """Set up results table containing mean, standard deviation and confidence interval
-    for each estimated parameter.
+def bootstrap_from_outcomes(base_outcome, bootstrap_outcomes):
```

Comment:
We need it still. Janos and I talked about this recently. The user should be able to call ```bootstrap_from_outcomes``` independently from the main ```bootstrap``` function. In cases such as passing previously computed ```bootstrapped_outcomes``` into ```bootstrap_from_outcomes``` directly rather than calling ```bootstrap``` and having ```bootstrap_outcomes``` recomputed

Reply:
Okay.